### ISSUE TITLE
Support Youtube thumbnail URLs

### ISSUE THREAD

**Issue by augbog** [regexps/youtube-regex#2]
Is there a way we can have it so this module supports Youtube thumbnail URLs? Here is a relevant [Stackoverflow question](http://stackoverflow.com/questions/2068344/how-do-i-get-a-youtube-video-thumbnail-from-the-youtube-api).

Ideally the regex would support URLs such as 

    https://img.youtube.com/vi/<insert-youtube-video-id-here>/default.jpg
    http://img.youtube.com/vi/<insert-youtube-video-id-here>/hqdefault.jpg

Along with the other quality thumbnails.

**Comment by tunnckoCore** [comment]
Hi and thanks for this issue. I will review it in the weekend.

**Comment by tunnckoCore** [comment]
@augbog hm, or it's pretty easy http://regexr.com/3bdh0 like this?

**Comment by augbog** [comment]
Yup that looks about right. I'm not sure what exactly the different urls are (all the ones listed in the Stackoverflow question are ones I go off of) and it seems to work fine. The regex you use for the youtube video id should be the same though in the middle part of the url.

Here's more info on [Video thumbnails](https://support.google.com/youtube/answer/72431?hl=en) if it helps!
